### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8228,6 +8232,8 @@ cursos->cursors, cursor,
 cursot->cursor
 cursro->cursor
 curvelinear->curvilinear
+cusstom->custom
+cusstoms->customs
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:8236]: If you don't mind adding these.
`cusstomer->customer`
`cusstomers->customers`
`cusstomizable->customizable`
`cusstomization->customization`
`cusstomize->customize`
`cusstomized->customized`

  * https://github.com/search?q=cusstomer&type=code
  * https://github.com/search?q=cusstomers&type=code
  * https://github.com/search?q=cusstomizable&type=code
  * https://github.com/search?q=cusstomization&type=code
  * https://github.com/search?q=cusstomize&type=code
  * https://github.com/search?q=cusstomized&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1236,6 +1236,10 @@ algorithemically->algorithmically
 algorithems->algorithms
 algorithic->algorithmic
 algorithically->algorithmically
+algorithim->algorithm
+algorithimes->algorithms
+algorithimical->algorithmical
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:1241]: The suggested word `algorithmical` is not found in major dictionaries (Eg. Oxford Learners, Merriam-Webster, etc).
If fact, the suggested word `algorithmical` itself is flagged as a misspell in `dictionary.txt`, this also explained the PR checks failure. You might want to change to `algorithimic->algorithmic`

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi can you re-review please? Thanks!

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**